content/docs/2.1/scalers/influxdb.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "InfluxDB"
+layout = "scaler"
+availability = "v2.1+"
+maintainer = "Community"
+description = "Scale applications based on InfluxDB queries"
++++
+
+### Trigger Specification
+
+This specification describes the `InfluxDB` trigger that scales based on the results of a flux query.
+
+```yaml
+triggers:
+  - type: influxdb
+    metadata:
+      serverURL: http://influxdb:8086
+      organizationName: influx-org
+      organizationNameFromEnv: INFLUXDB_ORG_NAME
+      thresholdValue: '4'
+      query: |
+        from(bucket: "bucket-of-interest")
+        |> range(start: -12h)
+        |> filter(fn: (r) => r._measurement == "stat")
+      authToken: some-auth-token
```

Comment:
Can we comment this one out, or at least mention that you either have to choose between `authToken` or `authTokenFromEnv` and not both please?